src/frontend/src/components/AppHeader/AppHeader.jsx

```diff
@@ -64,10 +75,34 @@ function AppHeader() {
                         <HeaderGlobalAction aria-label='Notifications'>
                             <Notification />
                         </HeaderGlobalAction>
-                        <HeaderGlobalAction aria-label={login}>
+                        <HeaderGlobalAction aria-label={login} onClick={togglePanel()}>
```

Comment:
We can simplify this and remove the toggle method by just calling the setter hook directly,

```suggestion
                        <HeaderGlobalAction aria-label={login} onClick={() => setUserPanel(!isUserPanel)}>
```